scripts/verify-agent-memory.sh

```
@@ -73,6 +78,49 @@ for required_contract in "${required_agent_contracts[@]}"; do
     fi
 done
 
+required_openrouter_contracts=(
+    '`OPENROUTER_API_KEY` is supplied by the environment.'
+    'if [[ -n "${OPENROUTER_API_KEY:-}" ]]; then'
+    'Run this check before asking Tim to create, paste, or reconfigure a key.'
+    'env -u OPENROUTER_API_KEY \'
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)